rector-ci.php

```diff
@@ -9,7 +9,6 @@
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
-    $containerConfigurator->import(__DIR__ . '/packages/EasyStandard/config/rector-set.php');
```

Comment:
This way we stop using our set in CI. I think we should introduce a new set for this (right in the monorepo) or simply move the contents of that set here.

Reply:
added to config, but rector checks are disabled in php-ci